students/ltrisolini/Lesson02/print_grid.py

```diff
@@ -0,0 +1,42 @@
+###Lesson 2 Grid Printer Exercise###
+
+###Values###
+c = "+"
+h = "-"
+v = "|"
+s = " "
+n = 4
+
+###Part 1###
+print (c + (h*n) + c + (h*n) + c )
+for i in range(n):
+   print(v + (s*n) + v + (s*n) + v)
+print (c + (h*n) + c + (h*n) + c )
+for i in range(n):
+   print(v + (s*n) + v + (s*n) + v)
+print (c + (h*n) + c + (h*n) + c )
```

Comment:
this code would be difficult to read.. while there are some cases when one letter vars are ok; in most scenarios it is best to name your vars that are more clear otherwise reader has to keep track what each letter means.

alternatively:
```
plus = '+'
dash = '-'
pipe = '|'
space = ' '
multiplier = 4

print(plus + dash * multiplier + plus + dash * multiplier + plus)
```
isn't that more readable?